src/elements/content-sidebar/SidebarFileProperties.js

```diff
@@ -55,7 +55,12 @@ const SidebarFileProperties = ({
                     : {}
             }
             size={getFileSize(file.size, intl.locale)}
-            uploader={getProp(file, 'created_by.name')}
+            // use uploader_display_name if uploaded anonymously
+            uploader={
+                getProp(file, 'created_by.id') === 2
```

Comment:
Is it string '2' or number 2? Could we reference a constant like https://github.com/box/box-ui-elements/blob/9fd8627c260146dbcb4e9441df068877f34f2d3c/src/constants.js#L382 to avoid magic numbers?

Reply:
addressed @ConradJChan